nuxeo-lambda-image-conversion/src/test/java/org/nuxeo/lambda/image/conversion/common/LambdaFeature.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * (C) Copyright 2006-2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Contributors:
+ *     anechaev
+ */
+package org.nuxeo.lambda.image.conversion.common;
+
+import org.nuxeo.ecm.platform.test.PlatformFeature;
+import static org.nuxeo.lambda.image.conversion.common.Constants.BUCKET_PREFIX_PROP;
+import static org.nuxeo.lambda.image.conversion.common.Constants.BUCKET_PROP;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.SimpleFeature;
+
+import java.util.Properties;
+
+@Features(PlatformFeature.class)
+@Deploy({ "org.nuxeo.lambda.core", "org.nuxeo.lambda.image.conversion", "org.nuxeo.ecm.platform.picture.api",
```

Comment:
Following [NXP-24366](https://jira.nuxeo.com/browse/NXP-24366), we can now express several `@Deploy` on the same entity. We have changed guidelines and now use several annotation instead of array.